--- a/src/cf.data.pre
+++ b/src/cf.data.pre
@@ -3516,6 +3516,19 @@
 			reference a PEM file containing both the certificate
 			and private key.
 	
+	Notes:
+
+	On Debian/Ubuntu systems a default snakeoil certificate is
+	available in /etc/ssl and users can set:
+
+		sslcert=/etc/ssl/certs/ssl-cert-snakeoil.pem
+
+	and
+
+		sslkey=/etc/ssl/private/ssl-cert-snakeoil.key
+
+	for testing.
+
 	sslcipher=...	The list of valid SSL ciphers to use when connecting
 			to this peer.
 
